plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/common/Field.tsx

```diff
@@ -0,0 +1,43 @@
+import { styled } from "@web/theme";
+
+type Props = {
+  title: string;
+  titleWidth?: number;
+  fieldGap?: number;
+  value: JSX.Element;
+};
+
+const Field: React.FC<Props> = ({ title, titleWidth, value }) => {
+  return (
+    <Wrapper>
+      <FieldTitle width={titleWidth}>{title}</FieldTitle>
+      <FieldValue>{value}</FieldValue>
+    </Wrapper>
+  );
+};
+
+export default Field;
+
+const Text = styled.p`
+  margin: 0;
+`;
+
+const Wrapper = styled.div<{ gap?: number }>`
+  display: flex;
+  align-items: center;
+  ${({ gap }) => gap && `gap: ${gap}px;`}
+  height: 32px;
+`;
```

Comment:
Thanks for making these common components. If I get time to refactor my field components it'll be nice to use these.
There is code here and there though that has unused/unneeded bits though. For example, this Wrapper stopped using the gap prop at some point, so please remove it.

Reply:
Done!